logstash-core/lib/logstash/pipeline.rb

```diff
@@ -56,6 +56,7 @@ def self.validate_config(config_str, settings = {})
     begin
       # There should be a better way to test this
       self.new(config_str, settings)
+      true
     rescue => e
       e.message
```

Comment:
Consider using nil instead of true so that the test on line 172 becomes so much more readable, nil being the absence of an error.  This is similar to the way Ruby returns nil on compact! in the absence of a change.
```ruby
if config_error
  @logger.fatal I18n.t("logstash.error", :error => config_error)
  return 1
else
  @logger.terminal "Configuration OK"
  return 0
end
```